code/game/machinery/doors/door.dm

```diff
@@ -78,6 +78,9 @@ var/list/all_doors = list()
 	return TRUE
 
 /obj/machinery/door/Bumped(atom/AM)
+	if (panel_open)
```

Comment:
I think this is wrong but I didn't test it, wouldn't this prevent you from walking into airlocks to open them if their panel is open?
That doesn't sound good.
Currently, `/obj/machinery/door/Bumped` in doors/doors.dm:80 does not call parent so this check on airlock.dm:388 never runs.

Reply:
I think you're right, I mixed up in this one. Since door.dm never calls `..()` the other proc was never being called. Good catch